src/routes/assistants/+page.svelte

```diff
@@ -25,6 +27,19 @@
 		}
 		goto(newUrl);
 	};
+
+	function getHref(keysAdd: { key: string; val: string }[], keysDelete?: string[]) {
```

Comment:
todo: should take the url as argument like in https://github.com/huggingface/chat-ui/commit/8cd7b98d4fa4ceb16f649714fc8d56d811a9c39d

(not for myself. I should refactor `getHref` as a utility function that can be used in different components)

Reply:
handled in https://github.com/huggingface/chat-ui/pull/773/commits/291a9be2122cfbf7b08dd94b9e432c5be754abc7 & https://github.com/huggingface/chat-ui/pull/773/commits/941380bacf6878a8e58b75e937bb015e8b7d170f